README

```diff
@@ -3,6 +3,9 @@
 Godef, given an expression or a location in a source file, prints the
 location of the definition of the symbol referred to.
 
+Installation:
+go get github.com/rogpeppe/godef
```

Comment:
Good suggestion, you may wanna add `-u` as well and put it between ```s like below

```
go get -u github.com/rogpeppe/godef
```
This will be easier to copy and it will update the tool if it already exists.

Reply:
It's not markdown, so ``` won't do anything, but I'll update with the flag